pkg/spire/spire.go

```diff
@@ -254,3 +284,206 @@ func (sc *SpireServerApiClient) Close() {
 		// Log error
 	}
 }
+
+func (sc *SpireServerApiClient) fetchSVID() (*x509svid.SVID, error) {
+	if sc.SVID != nil {
+		return sc.SVID, nil
```

Comment:
I'm not sure if this is valid since the SVID may expire, I believe the workload API has some utilities to take advantage of caching already.

Reply:
removed storing SVID.